### PR TITLE
adding an action

### PR DESCRIPTION
**Pull request by 18620146591**
增加了一个action
这个action叫detail_mining
可让LLM针对特定的”##讨论目标“和”##讨论记录“，进一步挖掘值得关注的细节，从而深化讨论。
微信图片_20230912174221

### CODE REVIEW

File: metagpt/actions/detail_mining.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+##
+"""
+OUTPUT_MAPPING = {
+    "OUTPUT": (str, ...),
+}
+
+
+class DetailMining(Action):
+    def __init__(self, name="", context=None, llm=None):
```
Comment (stellaHSR) [inline metagpt/actions/detail_mining.py:44]: You can add a brief description here, for example, to provide more detail when given a clear goal and context.

File: metagpt/actions/detail_mining.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        super().__init__(name, context, llm)
+
+    async def run(self, topic, record) -> ActionOutput:
+
+        prompt = PROMPT_TEMPLATE.format(topic,record,format_example=FORMAT_EXAMPLE)
```
Comment (stellaHSR) [inline metagpt/actions/detail_mining.py:49]: follow PEP8

File: metagpt/actions/detail_mining.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2023/9/12 17:45
+@Author  : fisherdeng
+@File    : detail_mining.py
+"""
+
```
Comment (stellaHSR) [inline metagpt/actions/detail_mining.py:8]: rm this line

File: metagpt/actions/detail_mining.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+{record}
+
+##Format example
+{format_example}
+-----
+
+任务: 参考 ##讨论目标 和 ##讨论记录 进一步询问你感兴趣的细节，字数不多于150字
+特别注意1：你只是单纯地询问，不要赞同或否定任何人的任何观点
+特别注意2:本次输出,仅包含 ##OUTPUT 这一主题，不要增加、减少或改变主题。输出时，以'##OUTPUT'开头，然后马上换行，再正式输出内容，仔细参考"##Format example"中的格式。
+"""
```
Comment (stellaHSR) [inline metagpt/actions/detail_mining.py:26]: Can you also provide an Engilish version?

**Comment by stellaHSR** [comment]
Also, provide a unit test for this action pls.

**Comment by stellaHSR** [comment]
LGTM

**Comment by stellaHSR** [comment]
LGTM

**Review by sablin39 — APPROVED** [review]

**Pull request merged**